main.tf

```diff
@@ -60,10 +60,28 @@ resource "aws_iam_role_policy_attachment" "lambda_policy_attachment" {
   role       = aws_iam_role.lambda_role.name
 }
 
+data "local_file" "lambda_handler" {
+  filename = "${local.lambda_dir}/handler.py"
+}
+
+data "local_file" "upsert_query" {
+  count    = var.upsert_query != "" ? 1 : 0
```

Comment:
Hva brukes denne til?

Reply:
Hvis `count = 0`, så blir det ikke laget noen data/ressurs (eller hva det heter).
...men mtp at vi uansett må gjøre en lignende sjekk nede i `"lambda_deploy_package"`, så er den vel egentlig veldig overflødig nå 🤔. Jeg fjerner denne, regner med oversiktligheten er bedre enn den eventuelle ressursbruken.

Reply:
Kan vi ikke bare ta inn en liste med filer som skal være del av lambdaen i stedet for? Da kan man sende inn hva som helst, og så bruke en for_each for å lage zip archive?